contracts/subsidy-types/src/lib.rs

```diff
@@ -0,0 +1,36 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+//! Types used for transactions which support subsidizing.
+
+#![no_std]
+#![deny(missing_docs)]
+#![deny(clippy::pedantic)]
+
+extern crate alloc;
+use alloc::vec::Vec;
+
+mod sig;
+
+pub use sig::*;
+
+use bls12_381_bls::{PublicKey, Signature};
+
+use rkyv::{Archive, Deserialize, Serialize};
+
+/// Subsidy a contract with a value.
+#[derive(Debug, Clone, PartialEq, Eq, Archive, Serialize, Deserialize)]
+#[archive_attr(derive(bytecheck::CheckBytes))]
+pub struct Subsidy {
+    /// Public key to which the subsidy will belong.
+    pub public_key: PublicKey,
```

Comment:
Shouldn't we refer to this as BlsPublicKey?

Reply:
it is a `bls12_381_bls::PublicKey`, not sure why should it be something else?

Reply:
subsidy-types crate has been removed